charts/ocis/values.yaml

```diff
@@ -170,8 +170,9 @@ features:
       # -- Authentication method for the SMTP communication. Possible values are ‘login’, ‘plain’, ‘crammd5’, ‘none’
       # If set to another value than `none`, a secret referenced by `notificationsSmtpSecretRef` needs to be present.
       authentication: none
-      # -- Encryption method for the SMTP communication. Possible values are ‘starttls’, ‘ssl’, ‘ssltls’, ‘tls’ and ‘none’.
-      encryption: none
+      # -- Encryption method for the SMTP communication. Possible values are ‘starttls’, ‘ssltls’, ‘auto’ and ‘none’.
+      # -- Values 'ssl' and 'tls' are deprecated and will be removed in version "6.0.0". Use 'starttls' instead of 'tls' and 'ssltls' instead of 'ssl'.
```

Comment:
`will be removed in version "6.0.0"`

I think this leaves some rooms for confusion, because this Helm Chart also has a version (that is far away from version 6) but we could improve it like this: `will be removed in oCIS release 6.0.0`